tests/unit/components/sl-calendar-test.js

```diff
@@ -11,32 +13,6 @@ moduleForComponent( 'sl-calendar', 'Unit | Component | sl calendar', {
     unit: true
 });
 
-test( 'Default class name is present', function( assert ) {
```

Comment:
The test is done indirectly in every single reference to this.$( '.sl-calendar' ) in the integration tests.